includes/class-wc-payments-fraud-service.php

```diff
@@ -188,4 +207,20 @@ private function get_cookie_session_id() {
 		$cookie_customer_id = $cookie[0];
 		return $wpcom_blog_id . '_' . $cookie_customer_id;
 	}
+
+	/**
+	 * If a "forterToken" cookie is present, send it to the WCPay server so the
+	 * current browsing session can be linked to the account. It will only be sent once.
+	 */
+	public function send_forter_cookie_token() {
```

Comment:
The linter is complaining about accessing this cookie directly:
```
 223 | ERROR | $_COOKIE data not unslashed before sanitization. Use
     |       | wp_unslash() or similar
 223 | ERROR | Detected usage of a non-sanitized input variable:
     |       | $_COOKIE['forterToken']
```

Reply:
I added an `ignore` comment in https://github.com/Automattic/woocommerce-payments/pull/1349/commits/08cdbca9aaf3ff921d8103a93bad142c50adac99 . We don't know what kind of contents that string has, so I'm not comfortable sanitizing it.

Reply:
Sounds good, thanks for adding that. Would you mind also adding a comment above that `ignore` explaining why we're not sanitizing it?